apps/meteor/server/startup/migrations/index.ts

```diff
@@ -35,4 +35,6 @@ import './v298';
 import './v299';
 import './v300';
 import './v301';
+import './v302';
+import './v303';
```

Comment:
You're importing two new files but only creating one new migration.

Reply:
Because the `v302` was not imported before. Should I ignore it, then?

Reply:
As it's unrelated to your PR you can ignore it. I'll figure out why it's there and fix it on a separate PR.